lib/worker.js

```diff
@@ -182,6 +184,7 @@ worker.prototype.perform = function(job, callback) {
                   if(returnCounter !== 3){
                     self.emit('failure', self.queue, job, callbackError);
                   }else{
+                    self.result = result;
```

Comment:
I don't think so... This is all internal API stuff, no one should be using it. Just make sure the emitter still works!